troposphere/static/js/components/modals/RequestMoreResourcesModal.react.js

```diff
@@ -101,7 +102,13 @@ define(
 
       renderBody: function () {
         var identities = stores.IdentityStore.getAll(),
-            instances = stores.InstanceStore.getAll();
+            instances = stores.InstanceStore.getAll(),
+            username = stores.ProfileStore.get().get('username'),
```

Comment:
It occurs to me I don't know which is better:
- `context.profile.get('username')`
- or, `stores.ProfileStore.get().get('username')` 

I guess this shows (yet) another awkwardness with the `get()` method on `stores`.